src/main/java/com/microsoft/sqlserver/jdbc/Parameter.java

```diff
@@ -296,7 +296,7 @@ void setValue(JDBCType jdbcType, Object value, JavaType javaType, StreamSetterAr
         }
 
         // forceEncryption is true, shouldhonorae is false
-        if ((true == forceEncrypt) && (false == Util.shouldHonorAEForParameters(stmtColumnEncriptionSetting, con))) {
+        if ((forceEncrypt) && (!Util.shouldHonorAEForParameters(stmtColumnEncriptionSetting, con))) {
```

Comment:
Unnecessary brackets around both `forceEncrypt` and `!Util.shouldHonorAEForParameters(stmtColumnEncriptionSetting, con)`. It seems the variable also has a typo as it should be `stmtColumnEncryptionSetting`, but the team will address this in another PR.